pkg/boot/universalpayload/handoffblock.go

```diff
@@ -0,0 +1,235 @@
+// Copyright 2024 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package universalpayload
+
+import (
+	"strings"
+	"unsafe"
+
+	guid "github.com/google/uuid"
+	"github.com/u-root/u-root/pkg/boot/kexec"
+)
+
+// Efi HOB Handoff Info Table
```

Comment:
I know Efi has their own capitalization rules, I think Go rules would prefer these to all be EFI, not Efi

Reply:
Cool beans